support_prometheus.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Setup Prometheus Monitoring on Chain
+  hosts: telemetry
+  vars_files:
```

Comment:
explicitly set it to gather_facts: false. Default is true.

Reply:
done.